test/unit/pnpm-symlinks/output.js

```diff
@@ -0,0 +1,11 @@
+[
+  "test/unit/pnpm-symlinks/input.js",
+  "test/unit/pnpm-symlinks/node_modules/.pnpm/parse5-htmlparser2-tree-adapter@6.0.1/node_modules/parse5",
```

Comment:
Is this an expected behavior that includes symlink directories as NodeFileTraceResult?